src/components/ChallengeSearchResult.vue

```diff
@@ -33,6 +33,11 @@
                             {{
                                 new Date(challenge.endDate).toLocaleDateString(
                                     "nl-nl",
+                                    {
+                                        day: "numeric",
```

Comment:
dus het komt er nu in te staan als: 30 november 2023?

Reply:
Ja

Reply:
beter